### ISSUE TITLE
[TB] Compliance Testing

### ISSUE THREAD

**Issue by dscamahorn** [finos/a11y-theme-builder#466]
## Suggestion/Concern
The ThemeBuilder outputs lack a third-party verification step for ensuring accessibility requirements have indeed been met.

## Proposed Solution
Test prototypes & solutions developed utilizing [Fable testing tool](https://makeitfable.com/). This would include adding a sample app that can be used for applying themes and then testing for verifying compliance.
